nomad/structs/workload_id_test.go

```diff
@@ -20,6 +20,7 @@ func TestNewIdentityClaims(t *testing.T) {
 
 	job := &Job{
 		ID:        "job",
+		ParentID:  "parentJob",
```

Comment:
the new helper method is simple enough, but is there also test coverage for parentless (service/system) jobs?

Reply:
Not of this function specifically, but most of the downstream code doesn't test with batch workloads (which is how this slipped thru in the first place).